NewForce-Cohort-5/tabloidmvc-the-brogrammers
# Ticket #9 View All Categories | Complete

# Description

This PR addresses the needs of Ticket 9, View All Categories.
- a Categories link has been added to the NavBar
- Clicking the Categories link will route the user to a table of all Categories currently available
- a Create New button exists, but does not yet function

## Type of change


-added CategoriesController.cs
-added Category/Index.cshtml (List)

# Testing Instructions

- from your terminal, git checkout into branch spcategories
- working in spcategories, run the application
- Check to ensure that the Categories link is present and functional
- check the Categories table to ensure that 13 pre-seeded items are present
- check all existing application functions for any issues

# Checklist:
